src/redist/targets/BundledManifests.targets

```
@@ -60,10 +66,6 @@
     </GenerateGuidFromName>
       
     <ItemGroup>
-      <BundledManifests Update="@(BundledManifests)" >
-        <RestoredNupkgContentPath>$(NuGetPackageRoot)%(Identity)/%(Version)</RestoredNupkgContentPath>
-      </BundledManifests>
-      
       <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\**\*.json;%(BundledManifests.RestoredNupkgContentPath)\**\*.targets"
                        DestinationPath="%(WorkloadManifestId)/"/>
     </ItemGroup>
```

One pattern I've found helpful is to error if globbing returns an empty sequence. Something like:
```suggestion
    </ItemGroup>
    
    <Error Text="No BundledManifests found." Condition="'@(ManifestContent->Count())' == '0'" />
```